calicoctl/commands/get.go

```diff
@@ -66,53 +62,60 @@ Options:
 		return nil
 	}
 
-	cmd := get{}
-	results := executeConfigCommand(parsedArgs, cmd)
-	glog.V(2).Infof("results: %+v", results)
-
-	if results.err != nil {
-		fmt.Printf("Error getting resources: %v\n", results.err)
-		return err
+	var rp resourcePrinter
+	output := parsedArgs["--output"].(string)
+	switch output {
+	case "yaml":
+		rp = resourcePrinterYAML{}
+	case "json":
+		rp = resourcePrinterJSON{}
+	case "ps":
+		rp = resourcePrinterTable{wide: false}
+	case "wide":
+		rp = resourcePrinterTable{wide: true}
+	default:
+		// Output format may be a key=value pair, so split on "=" to find out.  Pull
+		// out the key and value, and split the value by "," as some options allow
+		// a multiple-valued value.
+		outputParms := strings.SplitN(output, "=", 2)
+		outputKey := outputParms[0]
+		outputValue := ""
+		outputValues := []string{}
+		if len(outputParms) == 2 {
+			outputValue = outputParms[1]
+			outputValues = strings.Split(outputValue, ",")
+		}
+
+		switch outputKey {
+		case "go-template":
+			if outputValue == "" {
```

Comment:
This doesn't validate against the case that someone puts an `outputValue` that has commas in it (meaning multiple output values).  Not sure if we care so much... but you could probably do this instead:

```
if len(outputValues) != 1 {
  ...
}
```